makefiles/pseudomodules.inc.mk

```diff
@@ -191,6 +191,7 @@ PSEUDOMODULES += posix_headers
 PSEUDOMODULES += printf_float
 PSEUDOMODULES += prng
 PSEUDOMODULES += prng_%
+PSEUDOMODULES += psa_riot_cipher_%
```

Comment:
It's preferred to explicitly list all modules when possible.

Reply:
Done